### PR TITLE
FE & BE: Draft Page

### PR DESCRIPTION
**Pull request by Merannix**
1. Create draft inbox
    2. Hook up delete button for each row
    3. By default having draftStatus = false. Only update this to true when user clicks Save Draft

### CODE REVIEW

File: frontend/src/components/Pages/DraftPage/DraftPage.js [hunk 1]
```diff
@@ -1,20 +1,42 @@
...
     setIsPopupOpen(false);
+    setDraftToDelete(null); // Reset draft to delete
   };
 
-  const handleConfirmDelete = () => {
-    console.log("Draft deleted!");
+  const handleConfirmDelete = async () => {
+    if (draftToDelete) {
+      try {
+        await axios.delete(`${process.env.REACT_APP_BACKEND_URL}/records/${draftToDelete}`);
```
Comment (mengzhuou) [inline frontend/src/components/Pages/DraftPage/DraftPage.js:20]: add to connector.js

File: frontend/src/components/Functions/Draft/DraftList.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+import DraftButton from '../../Button/DraftButton/DraftButton';
+import './DraftList.css';
+
+const DraftList = ({ openDeletePopup }) => {
+    const [drafts, setDrafts] = useState([]);
+
+    useEffect(() => {
+        const fetchDrafts = async () => {
+            try {
+                const response = await axios.get(`${process.env.REACT_APP_BACKEND_URL}/records/drafts`);
```
Comment (mengzhuou) [inline frontend/src/components/Functions/Draft/DraftList.js:12]: add to connector.js

File: frontend/src/components/Button/DraftButton/DraftButton.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+const formatDate = (dateString) => {
+    const date = new Date(dateString);
+    const month = String(date.getMonth() + 1).padStart(2, '0');
+    const day = String(date.getDate()).padStart(2, '0');
+    const hours = String(date.getHours()).padStart(2, '0');
+    const minutes = String(date.getMinutes()).padStart(2, '0');
+    return `${month}-${day} ${hours}:${minutes}`;
+};
+
```
Comment (mengzhuou) [inline frontend/src/components/Button/DraftButton/DraftButton.js:14]: Make a comment that this is draft row

**Review by mengzhuou — CHANGES_REQUESTED** [review]
1. adjust font style according to Figma
2. Date is too close to Trashcan Icon

**Comment by Merannix** [comment]
Current page:
image

**Comment by Merannix** [comment]
Small screen size:
image

**Pull request merged**